docs/track.rst

```diff
@@ -2326,6 +2326,41 @@ Meta-data
 
 This operation returns no meta-data.
 
+transform-stats
+~~~~~~~~~~~~~~~
+
+With the operation type ``transform-stats`` you can call the `transform stats API <https://www.elastic.co/guide/en/elasticsearch/reference/current/get-transform-stats.html>`_.
+
+Properties
+""""""""""
+
+* ``transform-id`` (mandatory): The id of the transform.
+* ``condition`` (optional, defaults to no condition): A structured object with the properties ``path`` and ``expected-value``. If the actual value returned by transform stats API is equal to the expected value at the provided path, this operation will return successfully. See below for an example how this can be used.
+
+In the following example the ``transform-stats`` operation will wait until all data has been processed::
+
+    {
+        "operation-type": "transform-stats",
+        "transform-id": "mytransform",
+        "condition": {
+            "path": "checkpointing.operations_behind",
+            "expected-value": null
+        },
+        "retry-until-success": true
+    }
+
+Throughput will be reported as number of completed ``transform-stats`` operations per second.
+
+This operation is :ref:`retryable <track_operations>`.
+
+Meta-data
+"""""""""
+
+* ``weight``: Always 1.
+* ``unit``: Always "ops".
+* ``success``: A boolean indicating whether the operation has succeeded.
+* ``transform-stats``: The stats for the given transform id.
```

Comment:
As we no longer return the `transform-stats` meta-data, we should remove the docs reference.